vuepress/docs/next/tutorials/devops/installation/azure-kubernetes-service/azure-install.md

```diff
@@ -165,8 +158,7 @@ quickstart-kc-server-deployment-7c9bc65744-g52nx     1/1     Running     0
 quickstart-server-deployment-55fcfc6b68-szvkl        0/3     Pending     0          10m
 ```
 
-Double check your availability zones. By default an Azure cluster will include nodes from multiple zones but it will not provision storage
-for all of those nodes so you can't deploy your application.
+Double check your availability zones. By default an Azure cluster will include nodes from multiple zones but it will not provision storage for all of those nodes so you can't deploy your application.
```

Comment:
maybe add commas here...

Reply:
By default an Azure cluster will include nodes from multiple zones, but it will not provision storage for those nodes so you can't deploy your application. --added comma and removed 'all of'